Makefile.Common

```diff
@@ -220,6 +221,11 @@ misspell-correction: $(TOOLS_BIN_DIR)/misspell
 moddownload:
 	$(GOCMD) mod download
 
+.PHONY: testifylint
+testifylint: $(TESTIFYLINT)
+	@echo "running $(TESTIFYLINT)"
+	@$(TESTIFYLINT) --enable-all --disable=compares,empty,error-is-as,expected-actual,float-compare,formatter,go-require,negative-positive,require-error,suite-dont-use-pkg,suite-subtest-run,useless-assert -fix ./...
```

Comment:
I would prefer not having `--fix` as part of the `lint` step since as a user, it isn't something I would expect.

I think it should be two seperate steps like:

```
testifylint: $(TESTIFYLINT)
testifylint: TESTIFYLINT_ARGS ?= 
    # ... cmd stuff here

testifylint-fix: 
    @$(MAKE) testifylint TESTIFYLINT_ARGS="--fix"
```